src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private String color;
+
     public String getRandomColor() {
-        return null;
+        color = Color.values()[new Random().nextInt(Color.values().length)].name();
+        return color;
```

Comment:
```suggestion
        return Color.values()[new Random().nextInt(Color.values().length)].name();;
```
Create a random variable and use it instead of new Random()